CONTRIBUTING.md

```diff
@@ -1,57 +1,196 @@
+Monday after a sprint is completed. This will require us to have everything
+intended to be ready for the release merged to the `develop` branch by the
+second Thursday in a sprint. We are allowing ourselves an extra day at the end
+to provide us with the bandwidth needed to address any issues, merge
+conflicts, etc.
+
+This release branch will be deployed to our `stage` environment for FEC folks
+to review until we are ready to do a production deployment, which we have
+agreed will be that Wednesday pending any issues found. When the branch is
+created we have agreed to let folks know in the appropriate Slack channels
+ahead of time to prevent surprises.
+
+We currently have automatic deployments take place whenever code is
+merged/pushed to the `develop` (to our `dev` space) or `release/*` (to our
+`stage` space) branches.
+
+We have also created a separate `feature` space in Cloud Foundry that we only
+manually deploy to as needed. This will enable us to work on and test features
+that may require a long lead time before merging into `develop`. It will also
+enable us to test this work without being clobbered by the frequent
+deployments made when work is merged into `develop`.
+
+When working in the `feature` space, we've agreed to ask in advance to make
+sure no one else is currently utilizing the space for their work.
+
+
+### Development-related tools
+Both teams have agreed to the following tenents around the use of our
+development-related tools:
+
+- Folks should feel free to branch or fork the code as they see fit; neither
+  workflow has major pros or cons over the other, and if anything forking is
+  encouraged to ensure that there are no issues for external folks who would
+  like to contribute to the repositories as well.
+- We are going to stick with using
+  [git-flow](http://danielkummer.github.io/git-flow-cheatsheet/) to manage our
+  work in git, mainly because our deployment hooks are tied to its workflow.
+  We are leaving ourselves the option to investigate other tools or git
+  workflows in the future.
+- Our current setup with Travis-CI, hound, and Codecov are working fine for
+  both teams; we will continue to use them as is until we find they are no
+  longer meeting our needs.
+- The project originally made use of some Selenium tests, but these were
+  difficult to maintain and didn't provide much value. The teams have agreed
+  to consider introducing them again or something similar, but with the goals
+  of keeping them light-weight and maintainable; a spider or script of sorts
+  might be produced to help with catching dead links.
+
+
+### Monitoring
+We currently have several monitoring tools in place:
+
+- New Relic, mainly for basic pings
+- Sentry, for catching more verbose error reports and logs
+- A custom script powered by Mandrill to e-mail us nightly status reports in
+  the `dev`, `stage`, and `prod` environments.
+
+In the future, we would also like to find a solution for monitoring the AWS
```

Comment:
I have alerts set up with Amazon for the RDS boxes, you can have them too @ccostino now that you have access. I don't think the legal team needs them since it is just the data stuff.